index1-2.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="utf-8">
+        <title>1-2</title>
+        <script>
+            let count = 0;
+            while (1){
+        let place = prompt('東京の首都は？');
+
+            if(place == '東京'){
```

Comment:
`===` を利用するようにしてほしいです。
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Strict_equality
```suggestion
            if(place === '東京'){
```